gemlite/triton_kernels/gemm_A16fWnO16f_int32packing.py

```diff
@@ -49,9 +49,14 @@ def get_gemm_config():
                                 )
     return _configs
 
+def get_closest_m(M):
+    allowed_m_values = [8, 16, 32, 64, 128]  # Add or modify as needed
+    return min(allowed_m_values, key=lambda x: abs(x - M))
+
+@triton.heuristics({'CLOSEST_M': lambda args: get_closest_m(args['M'])})
```

Comment:
I think the simplest and more general way would be:
```Python
@triton.heuristics(values={'CLOSEST_M': lambda args: 2 ** int(math.ceil(math.log2(args['M'])))})
@triton.autotune(
    configs = get_gemm_config(),
    key=['CLOSEST_M', 'N', 'K', 'group_size', 'W_nbits'],
    prune_configs_by={
        'early_config_prune': kernel_config_pruner,
    },
    warmup=200, 
    rep=50, #20 for faster tuning 
)
```

Reply:
Sounds good! Users can modify the lambda function based on their use case if needed.